package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Hey Tim, why is this file needed now?

Reply:
Oops - looks like that was included inappropriately. Deleting...